src/main/docker/disableWeakTlsAlgorithms.patch

```diff
@@ -16,18 +16,13 @@
 
 --- java.security
 +++ java.security
-@@ -624,7 +624,29 @@
- # Example:
+@@ -739,6 +739,26 @@
  #   jdk.tls.disabledAlgorithms=MD5, SSLv3, DSA, RSA keySize < 2048
- jdk.tls.disabledAlgorithms=SSLv3, RC4, DES, MD5withRSA, DH keySize < 1024, \
--    EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
--    include jdk.disabled.namedCurves
-+    EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
+ jdk.tls.disabledAlgorithms=SSLv3, TLSv1, TLSv1.1, RC4, DES, MD5withRSA, \
+     DH keySize < 1024, EC keySize < 224, 3DES_EDE_CBC, anon, NULL, \
 +    CAMELLIA_128_CBC, AES_256_CBC, AES_128_CBC, \
 +    DES40_CBC, RC4_40, CAMELLIA_256_CBC, DES_CBC, \
 +    SEED_CBC, RC4_56, RC4_128, IDEA_CBC, RC2_CBC_40, \
-+    TLSv1, \
```

Comment:
This does not look right, it appears that we are now allowing TLSv1, TLSv1.1. See attached old and new java.security files.
[java.security.new.txt](https://github.com/CAFapi/opensuse-java11-images/files/6531423/java.security.new.txt)
[java.security.old.txt](https://github.com/CAFapi/opensuse-java11-images/files/6531424/java.security.old.txt)

Reply:
Ok, but the files I attached to my comment above were pulled out from the containers. I got the one from the latest snapshot jre11 image and the other from a local build of this branch. When I compare the java.security files the one from this branch does not have TLSv1 and TLSv1.1 disabled.

Not 100% sure how this works but I think the patch file replaces the whole section from the java.security file so the stuff that is there will be removed and as such also needs replaced.

Reply:
I built the local images again after deleting the old ones. I still find the TLSv1, TLSv1.1 disabled in both the images(checked in java.security files). The new patch file does not replace the whole section. It is only appending additional algorithms that need to be disabled.

Both these algorithms are also present in the java.security.new.txt attached by you at line 740

Reply:
My mistake, I see the TLSv1 and TLSv1.1 have just moved further up in the file.